app/forms/hub/outbound_call_form.rb

```diff
@@ -0,0 +1,73 @@
+module Hub
+  class OutboundCallForm < Form
+    attr_accessor :user_phone_number, :client_phone_number
+    delegate :call_hub_client_path,
+      :call_hub_client_url,
+      :outbound_calls_webhook_path,
+      :outbound_calls_webhook_url, to: 'Rails.application.routes.url_helpers'
+
+    before_validation do
+      self.user_phone_number = PhoneParser.normalize(user_phone_number)
+      self.client_phone_number = PhoneParser.normalize(client_phone_number)
+    end
+
+    validates :user_phone_number, phone: true
+    validates :client_phone_number, phone: true
+
+    def initialize(attrs = {}, **kwargs)
+      @user = kwargs[:user]
+      @client = kwargs[:client]
+
+      self.user_phone_number = @user&.phone_number
+      self.client_phone_number = @client&.phone_number
+      super(attrs)
+    end
+
+    def call!
```

Comment:
Consider `.dial` (and consider no `!` since you are returning false if not valid rather than throwing if not valid)